### PR TITLE
4.5 Refactor CLO to generate Kibana resource instead of managing Kibana deployment

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-970

Waiting on https://issues.redhat.com/browse/LOG-594

### CODE REVIEW

File: modules/cluster-logging-management-state-changing.adoc [hunk 1]
```diff
@@ -9,7 +9,6 @@ You must set the operator to the _unmanaged_ state in order to modify the compon
 
 * the Curator CronJob, 
 * the Elasticsearch CR,
-* the Kibana Deployment,
```
Comment (ewolinetz) [inline modules/cluster-logging-management-state-changing.adoc]: the Kibana CR

File: modules/cluster-logging-management-state-changing-es.adoc [hunk 2]
```diff
@@ -5,7 +5,8 @@
 [id="cluster-logging-management-state-changing-es_{context}"]
 = Changing the Elasticsearch management state
 
-You must set the operator to the _unmanaged_ state in order to modify the Elasticsearch deployment files, which are managed by the Elasticsearch Operator. 
+You must set the operator to the _unmanaged_ state in order to modify the Elasticsearch and Kibana deployment files, 
+, which are managed by the Elasticsearch Operator.
```
Comment (ewolinetz) [inline modules/cluster-logging-management-state-changing-es.adoc]: extra ", "

**Comment by mburke5678** [comment]
@anpingli Very small change. Please take a look.

**Comment by mburke5678** [comment]
@anpingli Very small change. Please take a look.

**Pull request merged**